src/towncrier/newsfragments/357.feature.rst

```diff
@@ -0,0 +1 @@
+Handle deletion of uncommitted news fragments
```

Comment:
Maybe something like this.

```suggestion
It can now handle news fragments which are not part of the git repository. For example uncommited or unstagged files.
```